tests/src/core/parse.js

```diff
@@ -21,4 +22,37 @@ describe('parse(content, { settings, ecmaFeatures })', function () {
       .not.to.throw(Error)
   })
 
+  it('passes expected parserOptions to custom parser', function () {
+    const parseSpy = sinon.spy()
+    const parserOptions = { ecmaFeatures: { jsx: true } }
+    require('./parseStubParser').parse = parseSpy
```

Comment:
require has a cache, so you get the same value out of it every time - any reason this isn't required at the top level?